diagnostic.md

```diff
@@ -41,10 +47,13 @@ What selectors could we write to reference each of the following categories of e
 * The element matching ID `contentPane`
 * Only those elements belonging to both the `important` and `red` classes
 * All `p` elements whose parent elements are `div`s
-<!-- your answer starts here -->
-
-<!-- your answer ends here -->
+```CSS
```

Comment:
All elements that belong to the class big
The element matching ID contentPane
Any elements that belong both to the important and red classes
Any p elements whose parent elements are divs